File: python-backend/src/app/internal_dto/internal_interview_question_create_response.py
from pydantic import BaseModel, Field


class InternalInterviewQuestionCreateResponse(BaseModel):
    """Java 返回给 Python 的面试题创建结果。"""

    round_id: int = Field(..., alias="roundId", description="题目轮次记录ID")
    round_no: int = Field(..., alias="roundNo", description="第几轮问题，从1开始")

File: java-backend/common/src/main/java/com/airesumeforge/common/dto/interview/internal/response/InternalInterviewQuestionCreateResponse.java
package com.airesumeforge.common.dto.interview.internal.response;

import com.airesumeforge.common.dto.interview.response.InterviewOptionResponse;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * 面试题目创建响应
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class InternalInterviewQuestionCreateResponse {

    /**
     * 题目记录ID
     */
    private Long roundId;

    /**
     * 轮次编号
     */
    private Integer roundNo;
}